msgpack-core/src/test/scala/org/msgpack/core/InvalidDataReadTest.scala

```diff
@@ -0,0 +1,20 @@
+package org.msgpack.core
+
+/**
+ *
+ */
+class InvalidDataReadTest extends MessagePackSpec {
+
+  "Reading long EXT32" in {
+      val msgpack = createMessagePackData(p => p.packExtensionTypeHeader(MessagePack.Code.EXT32, Int.MaxValue))
+      val u = MessagePack.newDefaultUnpacker(msgpack)
+      try {
+        intercept[MessageInsufficientBufferException] {
```

Comment:
The expectation of this test case is unclear to me. I guess you want to avoid 2GB memory allocation in this test, though. How about adding a comment to describe your intention?

Reply:
Added more comment to explain why we need to use this a bit weird test code